### DISCUSSION TITLE
Lightburn files to Lasergrbl

### DISCUSSION THREAD

**Discussion by gepol2** [arkypita/LaserGRBL#2295] [Q&A]
Hello, how can I insert .lbrn2 files to Lasergrbl?

**Comment by arkypita** [comment]
Hello
Lightburn is a commercial, closed source, software. Their file format is proprietary, i don't know if they have never released a documentation of internal file format and even if, this file format is related to their software with its feature, that is not available in LaserGRBL. 

The only file that is common between both is the gcode (.nc or .gcode) that contains instruction for laser.
